src/main/java/de/uhd/ifi/se/decision/management/jira/consistency/implementation/BasicDuplicateTextDetector.java

```diff
@@ -0,0 +1,84 @@
+package de.uhd.ifi.se.decision.management.jira.consistency.implementation;
+
+import com.atlassian.jira.issue.Issue;
+import de.uhd.ifi.se.decision.management.jira.classification.preprocessing.Preprocessor;
+import de.uhd.ifi.se.decision.management.jira.consistency.DuplicateDetectionStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BasicDuplicateTextDetector implements DuplicateDetectionStrategy {
+
+	private static Preprocessor preprocessor;
+	private static final String fieldUsedForDetection = "DESCRIPTION";
+	private int minDuplicateLength;
+
+	public BasicDuplicateTextDetector(int minDuplicateLength){
+		if (preprocessor == null){
+			preprocessor = new Preprocessor();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5586048)